content/docs/handling-events.md

```diff
@@ -91,64 +91,64 @@ ReactDOM.render(
 
-If you aren't using class fields syntax, you can use an [arrow function](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions) in the callback:
+Om du inte använder klassfält-syntax så kan du använda en [pil-function](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions) i callbacken:
 
 ```js{7-9}
 class LoggingButton extends React.Component {
   handleClick() {
-    console.log('this is:', this);
+    console.log('this är:', this);
   }
 
   render() {
-    // This syntax ensures `this` is bound within handleClick
+    // Denna syntax säkerställer att `this` är bunden inuti handleClick
     return (
       <button onClick={(e) => this.handleClick(e)}>
-        Click me
+        Klicka
       </button>
     );
   }
 }
 ```
 
-The problem with this syntax is that a different callback is created each time the `LoggingButton` renders. In most cases, this is fine. However, if this callback is passed as a prop to lower components, those components might do an extra re-rendering. We generally recommend binding in the constructor or using the class fields syntax, to avoid this sort of performance problem.
+Problemet med den här syntaxen är att en annan callback skapas varje gång `LoggingButton` rendreras. I de flesta fall är det ok. Men om denna callback skickas som en prop till barn-komponenter, kan dessa komponenter råka göra en extra rendrering. Generellt sett rekommenderar vi att binda in konstruktorn eller använda klassfält-syntax för att undvika denna typ av prestandaproblem.
 
-## Passing Arguments to Event Handlers {#passing-arguments-to-event-handlers}
+## Passa argument till event-hanterare {#passing-arguments-to-event-handlers}
 
-Inside a loop it is common to want to pass an extra parameter to an event handler. For example, if `id` is the row ID, either of the following would work:
+Inuti en loop är det vanligt att man vill passa extra parametrar till en event hanterare. Till exempel om ´ìd` är radens ID skulle något av följande fungera:
 
 ```js
-<button onClick={(e) => this.deleteRow(id, e)}>Delete Row</button>
-<button onClick={this.deleteRow.bind(this, id)}>Delete Row</button>
+<button onClick={(e) => this.deleteRow(id, e)}>Ta bort rad</button>
+<button onClick={this.deleteRow.bind(this, id)}>Ta bort rad</button>
 ```
 
-The above two lines are equivalent, and use [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) and [`Function.prototype.bind`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind) respectively.
+Ovanstående två rader är likvärdiga och använder [pil-funktion](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) och [`Function.prototype.bind`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind).
```

Comment:
Use definitive article or plural for `arrow functions`

Reply:
@rloqvist Fixed